npm/vue/cypress.json

```diff
@@ -2,6 +2,10 @@
   "viewportWidth": 500,
   "viewportHeight": 500,
   "video": false,
+  "port": 4444,
+  "namespace": "__cypress-ct123",
+  "clientRoute": "/__ct/",
+  "socketIoRoute": "/__ct-socket.io",
```

Comment:
I don't understand why this is necessary.

Reply:
Please read the PR description coment and the code :)

Reply:
I did... a few times. The fact that these changes need to be in npm/vue/cypress.json instead of flags passed in circle.yml/package.json was confusing to me.

I get it now. It's not ideal, but it's fine for now.

Reply:
This config is internal and Only works using cypress.json. 

In ideal world I would like to not use cypress/vue but make separate amount of both failed and passed tests that we can use for testing.

Reply:
I have a fix for this... see `runner-ct/scripts/e2e.sh` that I committed onto this branch